sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/Readme.md

```diff
@@ -554,15 +554,15 @@ Use `Queries` in [TimeSeriesInsightsClient](https://github.com/Azure/azure-sdk-f
 - Computed values and the associated event timestamps by applying calculations defined by variables on raw events. These variables can be defined in either the Time Series Model or provided inline in the query.
 - Aggregated values and the associated interval timestamps by applying calculations defined by variables on raw events. These variables can be defined in either the Time Series Model or provided inline in the query.
 
-Response for the `Query` APIs are of type `QueryAnalyzer`. The QueryAnalyzer allows a developer to query for pages of results, while being able to perform operations on the result set as a whole. For example, to get a list of `TimeSeriesPoint` in pages, call the `GetResultsAsync` method on the `QueryAnalyzer` object. You can enumerate an AsyncPageable object using the `async foreach` loop.
+Response for the `Query` APIs are of type `TimeSeriesQuery`. The TimeSeriesQuery allows a developer to query for pages of results, while being able to perform operations on the result set as a whole. For example, to get a list of `TimeSeriesPoint` in pages, call the `GetResultsAsync` method on the `TimeSeriesQuery` object. You can enumerate an AsyncPageable object using the `async foreach` loop.
```

Comment:
Did we decide to go for a TimeSeriesQuery here? Or waiting for suggestions?

Reply:
Waiting on suggestions. I made this change before we talked about it offline :)

Reply:
I wouldn't check it in until we hear back from SDK board or leave this one alone for later.